client/components/Layout/LayoutCollectionHeader.tsx

```diff
@@ -17,10 +21,43 @@ type Props = {
 	content: LayoutBlockCollectionHeader['content'];
 };
 
+const MetadataDiv = (props: Props) => {
```

Comment:
Since this component doesn't render a `div`, I think there is room to name it more clearly. Maybe something like `MetadataList`, `MetadataFields`, `MetadataDict`, etc.